### DISCUSSION TITLE
auto crop before page content

### DISCUSSION THREAD

**Discussion by konos93** [ScanTailor-Advanced/scantailor-advanced#68] [General]
do u think that if we had something like auto-crop only the characters from page . and white everything around we would have better results . using something like yolo from computer vision or something . 

i am trying to use python , yolov8 to make an autocrop only for the characters of  page content and make a page number detector.

**Comment by darnn** [comment]
This is what Page Box in Select Content is for.

**Comment by konos93** [comment]
yeah but unfortunately many times i need to manually correct content box

**Comment by darnn** [comment]
Okay, but... I'm talking about the Page Box. It essentially does what you're talking about, just without altering the actual source file. Granted, if all your pages appear in different places in every file, you still have to adjust it manually, but this would be true of any outside solution as well, unless you can get YOLO (whatever it is) to identify a text area specifically and discard everything else, without discarding parts on the edges that you need. This is really the issue with actually modifying the file: When you're going over the result, you can only see what was left in, not what was cut out, so you can sometimes miss something that should have been left in but wasn't. 
Anyway, I really recommend you experiment a bit with the Page Box before you start bringing ML into this.
